pkg/persistence/config/internal/persistence/type_definition.go

```diff
@@ -322,7 +334,10 @@ func (c TypeDefinition) MarshalYAML() (interface{}, error) {
 				},
 			}, nil
 		}
-	}
 
+	case config.GrailFilterSegment:
+		return GrailFilterSegment, nil
```

Comment:
The cases above also check the FF, please do it here as well

Reply:
added